eventonica-api/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
```

Comment:
This is a nit, but do you know if you could change this to use `const`? I don't think you should need to use `var` at all with node on the backend.